<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommTab</class>
 <widget class="QWidget" name="CommTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>功能列表</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListWidget" name="listWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>自主控制</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>其他</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="dacPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>自主控制功能</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="title">
             <string>常规设置</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="4" column="3">
              <widget class="QCheckBox" name="wCheckBox_2">
               <property name="text">
                <string>写</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2" colspan="4">
              <widget class="QComboBox" name="ownerComboBox"/>
             </item>
             <item row="2" column="2">
              <widget class="QCheckBox" name="rCheckBox">
               <property name="text">
                <string>读</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>所有者(O):</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2" colspan="4">
              <widget class="QComboBox" name="groupComboBox"/>
             </item>
             <item row="0" column="5">
              <widget class="QPushButton" name="pushButton">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QCheckBox" name="rCheckBox_3">
               <property name="text">
                <string>读</string>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <widget class="QCheckBox" name="exeCheckBox">
               <property name="text">
                <string>执行</string>
               </property>
              </widget>
             </item>
             <item row="5" column="4">
              <widget class="QCheckBox" name="exeCheckBox_3">
               <property name="text">
                <string>执行</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>其他访问:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>访问:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QCheckBox" name="rCheckBox_2">
               <property name="text">
                <string>读</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>路径</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QCheckBox" name="wCheckBox">
               <property name="text">
                <string>写</string>
               </property>
              </widget>
             </item>
             <item row="5" column="3">
              <widget class="QCheckBox" name="wCheckBox_3">
               <property name="text">
                <string>写</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QCheckBox" name="subCheckBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>应用于子文件</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>组(G):</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2" colspan="3">
              <widget class="QLineEdit" name="lineEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="4">
              <widget class="QCheckBox" name="exeCheckBox_2">
               <property name="text">
                <string>执行</string>
               </property>
              </widget>
             </item>
             <item row="6" column="2" colspan="4">
              <widget class="QPushButton" name="setMaskButton">
               <property name="text">
                <string>保存全部</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>访问:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="isDirheckBox">
               <property name="text">
                <string>目录</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>70</height>
             </size>
            </property>
            <property name="title">
             <string>ACL设置</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QPushButton" name="setALCButton">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(241, 241, 241);</string>
               </property>
               <property name="text">
                <string>设置ACL</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="otherPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
